v1/api/swagger/components/schemas/Host.yaml

```diff
@@ -191,3 +192,11 @@ allOf:
         description: Host health summary status
       Labels:
         $ref: ./Labels.yaml
+      WWPNs:
+        type: array
+        description: FC host port world wide port names
```

Comment:
It should be FC HBA world wide port names. Make global change for all

Reply:
Ok.